src/actions.js

```diff
@@ -91,19 +91,22 @@ module.exports = {
 					label: 'Clip',
 					id: 'clip',
 					choices: [
-						...this.store.clips.map(({ name }, index) => ({ id: index + 1, label: `${index + 1}: ${name}` })),
 						{ id: 0, label: 'Select using custom variable' },
+						...[...Array(this.config.maxClips).keys()].map((index) => {
+							const name = this.store.clips[index]?.name || 'Clip ' + (index + 1)
+							return { id: index + 1, label: `${index + 1}: ${name}` }
+						}),
 					],
 					default: this.store.clips.length ? 1 : 0,
 				},
 				{
 					type: 'dropdown',
 					label: 'Custom Variable',
 					id: 'variable',
-					choices:
-						this.customVariables.length === 0
-							? [{ id: 0, label: 'No variables defined' }]
-							: Object.keys(this.customVariables).map((name, index) => ({ id: index, label: name })),
+					choices: [
+						{ id: 0, label: 'No variable selected' },
+						...Object.keys(this.customVariables).map((name, index) => ({ id: (index + 1), label: name })),
+					],
```

Comment:
This change is completely unrelated to your PR. This allows the use of custom/user-defined variables. This has nothing to do with the internal/module variables you're proposing. Please remove.

Reply:
Agree, this one slip through my commit to the branch (was not intended) and I will remove it. Nevertheless there should be a change in handling the custom variables. If you don't want to use a custom variable it will select one and the default should be not using a custom variable or am I wrong?